code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -95,6 +96,13 @@
 					findtarget()
 					return
 
+/obj/item/clothing/mask/facehugger/proc/spreadout()
+	if(!target && isturf(loc))
+		for(var/obj/item/clothing/mask/facehugger/F in loc)
+			if(F != src)
+				step(src, pick(1,2,4,8), 0)
```

Comment:
we have global list `cardinal` for this!

it's functionally the same but reads nicer, like defines